docs/src/index.md

```diff
@@ -16,17 +16,17 @@ FreeAgent(::AbstractString, ::Vector{<:AbstractAlgebraicAgent})
 
 To implement a custom algebraic agent type, you may want to use the convenience macro [`@aagent`](@ref) which supplies type fields expected (not required, though) by the interface.
 
-Next step is to implement the required interface functions:
+Next step is to implement the mandatory interface functions:
 
 ```@docs
 AlgebraicAgents._step!(::AbstractAlgebraicAgent)
 AlgebraicAgents._projected_to(::AbstractAlgebraicAgent)
-AlgebraicAgents.getobservable(::AbstractAlgebraicAgent, ::Any)
 ```
 
 For a deeper integration of the agent type, you may specialize the following functions:
 
 ```@docs
```

Comment:
should `observables` also be added to this list?

Reply:
Yes, thank you for pointing that out. I made a couple of additional changes: to get rid of macros whenever possible, I turned `@wrap` into a function `wrap_system`, and similarly for `@get_agent`, which is now called `extract_agent` (looking forward to any renaming).